lib/content._coffee

```diff
@@ -6,21 +6,18 @@ redis = require 'redis'
 URL = require 'url'
 Worker = require './worker'
 
+pkgcloud = require('pkgcloud')
+cloudClient = pkgcloud.storage.createClient
```

Comment:
Also, a minor code style thing: it's good to keep all the `require`s together up top (and we keep them in alphabetical order since the lists can get long), so move just the `pkgcloud = ` line to the previous block of `require`s.

(Also minor: no parens needed there.)